### DISCUSSION TITLE
Rendering multiple POIs at the same location more consistently

### DISCUSSION THREAD

**Discussion by JohannesDeml** [organicmaps/organicmaps#4181] [Ideas]
# Problem
I really enjoy using OM on my iPhone, but I stumbled into rendering problems of POIs, where I can only see one entry, even when there are multiple ones. I hope I didn't miss any other discussions, that already talk about this problem. Here are two examples:

## One Entry is shown
* https://omaps.app/44LUON8Z27/Garten
* om://44LUON8Z27/Garten

| View Mode  | Edit Mode | 
|:----:|:----:|
| ![signal-2023-01-01-173526_002](https://user-images.githubusercontent.com/4518980/210178919-9a1528fc-8c90-4154-b6c4-d7cd5a35c698.jpeg) | ![signal-2023-01-01-173526_003](https://user-images.githubusercontent.com/4518980/210178926-68b7cb9a-46ae-4ada-9e5d-58ae9904cd1c.jpeg) |
| Shows only one entry | There are actually 4 POIs at this address |

## Zooming changes POI
* https://omaps.app/44LUOONoJo/Grützmacher%2C_Ten_Hoff
* om://44LUOONoJo/Grützmacher%2C_Ten_Hoff

| Zooming changes the POI  |  
|:----:|
| <video src='https://user-images.githubusercontent.com/4518980/210178973-4358ff58-2330-46b7-b86b-122bed20d8c9.mp4' width=180/> |

# Solution
I think it could help for the first issue to allow to zoom in further to be able to render all POIs? For the second issue, I guess in this very case it would be cool to render them just side by side, since there is enough space. Otherwise, I'm not sure if garden should be rendered, if there is another POI, since the green area already gives this information (at least in cities).

**Comment by biodranik** [comment]
There are already several issues about it with some ideas. Let's aggregate them together )
